ui/src/app/pages/login/sign-in.spec.tsx

```diff
@@ -117,11 +117,40 @@ describe('SignInReact', () => {
     wrapper.find(AccountCreationTos).props().onComplete();
 
     expect(wrapper.exists(AccountCreation)).toBeTruthy();
-    wrapper.find(AccountCreation).props().onComplete(getEmptyProfile());
+    wrapper.find(AccountCreation).props().onComplete(createEmptyProfile(requireInstitutionalVerification));
 
     // Account Creation Survey (e.g. demographics) is part of the new-style flow.
     expect(wrapper.exists(AccountCreationSurvey)).toBeTruthy();
-    wrapper.find(AccountCreationSurvey).props().onComplete(getEmptyProfile());
+    wrapper.find(AccountCreationSurvey).props().onComplete(createEmptyProfile(requireInstitutionalVerification));
+
+    expect(wrapper.exists(AccountCreationSuccess)).toBeTruthy();
+  });
+
+  it('should handle sign-up flow for new account creation with institutional verification', () => {
+    props.serverConfig = {...defaultConfig, enableNewAccountCreation: true};
+    const requireInstitutionalVerification = true;
+
+    const wrapper = shallowComponent();
+
+    // To start, the landing page / login component should be shown.
+    expect(wrapper.exists(LoginReactComponent)).toBeTruthy();
+    // Simulate the "create account" button being clicked by firing the callback method.
+    wrapper.find(LoginReactComponent).props().onCreateAccount();
```

Comment:
why can't we just click the button?

Reply:
I don't know.  This is just copy/paste of the above test with a boolean flipped